canvas/cli/common.py

```diff
@@ -1,16 +1,48 @@
 import sys
+import json
 
 # keys: [(items key, title, pretty title), ...]
 def cli_list(items, keys, table = False, skip_headers = False,
```

Comment:
You're general model/plan here seems seems a bit off.
I would just change the main printing loop here to just collect the clean values in a line instead of printing them.
Then after that loop you can just have a clean conditional block for if it is a table, json, or standard.